index.bs

```diff
@@ -1516,6 +1527,8 @@ When this operation is invoked, the authenticator must perform the following pro
     so, return an error code equivalent to "{{NotAllowedError}}" and terminate the operation.
 - If the |requireResidentKey| flag is set to |true| and the authenticator cannot store a [=Client-side-resident Credential
     Private Key=], return an error code equivalent to "{{ConstraintError}}" and terminate the operation.
+- If the |requireRateLimiting| flag is set to |true| and the authenticator cannot support [=Rate Limiting=], return an error code 
```

Comment:
@equalsJeffH 

Another option is to add the rate limiting requirement in the definition of User Verification in Section 3.  The current definition already discusses security requirements, so adding a statement on brute force attack prevention does not seem excessive.  If you are OK with such an approach, I can close this PR and create a new one with the modified definition.